src/experiments.js

```diff
@@ -135,6 +135,23 @@ export function experimentToggles(win) {
     }
   }
 
+  // Read document level override from meta tag.
+  if (win.AMP_CONFIG
+      && Array.isArray(win.AMP_CONFIG['allow-doc-opt-in'])
+      && win.AMP_CONFIG['allow-doc-opt-in'].length > 0) {
+    const allowed = win.AMP_CONFIG['allow-doc-opt-in'];
+    const meta =
+        win.document.head.querySelector('meta[name="amp-experiments-opt-in"]');
+    if (meta) {
+      const optedInExperiments = meta.getAttribute('content').split(',');
```

Comment:
do we have the precedence order documented somewhere? (i know there was a discussion in a github issue but would be nice if we had official docs)